frontend/src/devicelist/__tests__/DeviceList.test.jsx

```diff
@@ -28,9 +28,18 @@ function verifyStaticElements() {
   expect(receiverTab.textContent).toBe("Receivers");
 }
 
+function nextTablePage() {
+  act(() => {
+    const nextPageButton = document.querySelector(
+      'span[title="Next Page"] button'
+    );
+    ReactTestUtils.Simulate.click(nextPageButton);
+  });
+}
+
 let container = null;
 let sampleSenders = null;
-let sampleReceivers = null;
+// let sampleReceivers = null;
```

Comment:
keeping receivers stuff commented so we don't forget them in the test re-write coming up?

Reply:
Yeah just commenting out all stuff that was necessary for the old tests to work.